source/docs/reference/environment-variables.md

```diff
@@ -0,0 +1,99 @@
+---
+id: environment-variables
+title: Environment variables
+---
+
+The environment variables available in Front-Commerce are the configurations that are likely to change depending on the current environment of your application. For instance, you could have three different environments: production, staging and local.
+
+These environment variables are the ones available on your server. You can also set them in your `.env` file in your root folder, and they will be loaded on launch.
+
+## Front-Commerce related variables
+
+### Host
+
+Configure the execution environment of the Front-Commerce's application:
+
+- `FRONT_COMMERCE_PORT` (default: 4000): The port of the launched server
+- `FRONT_COMMERCE_HOST` (default: 0.0.0.0): The host of the launched server. It might be useful to set it to 127.0.0.1 if you want to only listen local requests.
```

Comment:
```suggestion
- `FRONT_COMMERCE_HOST` (default: `0.0.0.0`): The host of the launched server. It might be useful to set it to `127.0.0.1` if you want to only listen local requests.
```